_rules/SC1-1-1-image-accessible-name-is-placeholder-text.md

```diff
@@ -0,0 +1,78 @@
+---
+name: Image accessible name is placeholder text
+description: |
+  This rule checks that image elements that use generic placeholder text as their accessible name do so without loss of infomation to the user.
+
+success_criterion:
+- 1.1.1 # Non-text Content
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Bryn Anderson
+---
+
+### Applicability
+
+The rule applies to HTML `input type="image"` or `area` elements, or any HTML element with the semantic role of `img`, where the element is [exposed to assistive technologies](#exposed-to-assistive-technologies) and has an [accessible name](#accessible-name) that exclusively uses [generic placeholder text](#generic-placeholder-text).
+
+### Expectation
+
+Each test target has an accessible name that can be used in place of the [non-text content](https://www.w3.org/TR/WCAG21/#dfn-non-text-content) of the element without loss of information to the user.
+
+## Assumptions
+
+*There are currently no assumptions.*
+
+## Accessibility support
+
+*There are no major accessibility support issues known for this rule.*
+
+## Background
+
+- [https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)
+- [Failure of Success Criterion 1.1.1 and 1.2.1 due to using text alternatives that are not alternatives (e.g., filenames or placeholder text)](https://www.w3.org/TR/WCAG20-TECHS/F30.html)
+
+## Test Cases
+
+### Pass example 1
+
+The element's accessible name uses generic placeholder text which accurately describes the image.
+
+```html
+<img alt="oage"> (need a source file)
```

Comment:
How can we judge that this is generic placeholder text without the surrounding context?